src/test/java/ca/wolfram/beta/symath/MathUtilsTest.java

```diff
@@ -0,0 +1,21 @@
+package ca.wolfram.beta.symath;
```

Comment:
We have travis though, what's wrong with looking at the debug message? It shows you on the log. I usually split tests based on what I'm testing. If I test one method with many cases I keep those in one method